File: Contract.Architecture.Frontends/Contract.Architecture.Web.Core/src/app/model/gegoennter-kundenstamm/gegoennte-kunden/dtos/i-gegoennter-kunde-create.ts
import { ApiGegoennterKundeCreate } from './api/api-gegoennter-kunde-create';

export interface IGegoennterKundeCreate {
    name: string;
    gegoennterName: string;
    gegoennterBoolean?: boolean;
    gegoennterDateTime?: Date;
    gegoennterDouble?: number;
    gegoennterGuid?: string;
    gegoennterInteger?: number;
    besteBankId: string;
}

export abstract class GegoennterKundeCreate {
    public static toApiGegoennterKundeCreate(iGegoennterKundeCreate: IGegoennterKundeCreate): ApiGegoennterKundeCreate {
        return {
            name: iGegoennterKundeCreate.name,
            gegoennterName: iGegoennterKundeCreate.gegoennterName,
            gegoennterBoolean: iGegoennterKundeCreate.gegoennterBoolean,
            gegoennterDateTime: iGegoennterKundeCreate.gegoennterDateTime,
            gegoennterDouble: iGegoennterKundeCreate.gegoennterDouble,
            gegoennterGuid: iGegoennterKundeCreate.gegoennterGuid,
            gegoennterInteger: iGegoennterKundeCreate.gegoennterInteger,
            besteBankId: iGegoennterKundeCreate.besteBankId,
        };
    }
}

File: Contract.Architecture.Backends/Contract.Architecture.Backend.Core/Contract/Logic/Modules/GegoennterKundenstamm/GegoennteKunden/DTOs/IGegoennterKundeCreate.cs
using System;

namespace Contract.Architecture.Backend.Core.Contract.Logic.Modules.GegoennterKundenstamm.GegoennteKunden
{
    public interface IGegoennterKundeCreate
    {
        string Name { get; set; }

        string GegoennterName { get; set; }

        bool? GegoennterBoolean { get; set; }

        DateTime? GegoennterDateTime { get; set; }

        double? GegoennterDouble { get; set; }

        Guid? GegoennterGuid { get; set; }

        int? GegoennterInteger { get; set; }

        Guid BesteBankId { get; set; }
    }
}